src/pages/register-a-beacon/beacon-information.tsx

```diff
@@ -21,8 +21,29 @@ import { Input } from "../../components/Input";
 import { InsetText } from "../../components/InsetText";
 import { Layout } from "../../components/Layout";
 import { IfYouNeedHelp } from "../../components/Mca";
+import { FormControl } from "../../lib/form/formControl";
+import { FormGroupControl } from "../../lib/form/formGroupControl";
+import { CacheEntry } from "../../lib/formCache";
 import { handlePageRequest } from "../../lib/handlePageRequest";
 
+const getFormGroup = ({
+  manufacturerSerialNumber,
+  beaconCHKCode,
+  beaconBatteryExpiryDateMonth,
+  beaconBatteryExpiryDateYear,
+  lastServicedDateMonth,
+  lastServicedDateYear,
+}: CacheEntry): FormGroupControl => {
+  return new FormGroupControl({
+    manufacturerSerialNumber: new FormControl(manufacturerSerialNumber),
+    beaconCHKCode: new FormControl(beaconCHKCode),
+    beaconBatteryExpiryDateMonth: new FormControl(beaconBatteryExpiryDateMonth),
+    beaconBatteryExpiryDateYear: new FormControl(beaconBatteryExpiryDateYear),
+    lastServicedDateMonth: new FormControl(lastServicedDateMonth),
+    lastServicedDateYear: new FormControl(lastServicedDateYear),
+  });
+};
+
```

Comment:
Good question.  I had put this in here as I needed to pass it to the `handlePageRequest` but this will actually need updating anyway as `manufacturerSerialNumber` is required and month/year need to be integers and limited to 2/4 characters respectively.

Point though that, yes, you could absolutely not use it and just use the `formData` object but I think from a consistency point with how all the other pages have been done it would be clearer to define it if it's a form and we are redirecting to the new next. Currently, we actually don't have any pages without validation so it would be an edge-case rather than a thing we would do regularly.

For the check answers page, I wouldn't expect us to create a `FormGroupControl`/`FormControl` classes as it isn't a form, it's just displaying the values.